tests/jobs/test_scheduler_job.py

```diff
@@ -1057,6 +1057,31 @@ def test_process_file_should_failure_callback(self):
             self.assertEqual("Callback fired", content)
             os.remove(callback_file.name)
 
+    def test_should_parse_only_active_dags(self):
```

Comment:
```suggestion
    def test_should_parse_only_unpaused_dags(self):
```

Active is another concept (the is_active column).

Reply:
:+1:  from me after this change is made

Reply:
Updated.